sklearn/cluster/hierarchical.py

```diff
@@ -783,9 +796,11 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         By default, no caching is done. If a string is given, it is the
         path to the caching directory.
 
-    n_components : int, optional
```

Comment:
Not important but we usually remove them when deprecating. No-one should start using them. I think the case that someone needs to look up a deprecated parameter are really rare.